src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -661,7 +661,7 @@ private JabRefPreferences() {
         // Currently, JabRef does not escape them
         defaults.put(KEY_GEN_FIRST_LETTER_A, Boolean.TRUE);
         defaults.put(KEY_GEN_ALWAYS_ADD_LETTER, Boolean.FALSE);
-        defaults.put(EMAIL_SUBJECT, Localization.lang("References"));
+        defaults.put(EMAIL_SUBJECT, "References");
```

Comment:
Idea: since we now loose our nice localization tests that make sure that "References" should be translated, what do you think about adding a `Localization.plan` method that simply returns the string. Than one could change the localization tests to also look for `plain` in addition to `lang` . That would minimize the disadvantages even more I think.

Reply:
Wouldn't be `Localization.plain("References")` then always the same as "References"?

Reply:
Yes. The point was to still have the localization tests which could look for `Localization.plain`